docs/source/api/tracker.md

```diff
@@ -86,21 +86,52 @@ a `Tracker.withComputation` call.
 
 ```javascript
 Tracker.autorun(async function example1(computation) {
-  let asyncData = 
-    await Tracker.withComputation(computation, () => asyncDataFunction());
-  let users = Meteor.users.find({}).fetch();
+  let asyncData = await asyncDataFunction();
+  let users =
+    await Tracker.withComputation(computation, () => Meteor.users.find({}).fetch());
 });
 ```
 > If you want to get computation in other way you can use `Tracker.currentComputation`
 
-#### Using async callbacks in versions of Meteor prior to 2.10
+
+{% apibox "Tracker.withComputation" %}
+
+In general, the rules to use `Tracker.withComputation` like this:
+1. `async` function *before the first* `await` if just like a sync one.
+2. `async` function *after the first* `await` looses the `Tracker.currentComputation`
+but it can be restored using `Tracker.withComputation`, *but only within the callback*.
+
+If you have for example:
+
+```javascript
+Tracker.autorun(async function (computation) {
+  let asyncData = await someAsyncCall();
+  let links = await LinksCollection.find({}).fetch(); // it will not trigger reruns.
+});
+```
+You can make this example reactive by wrapping the `Meteor.users.find` call in a `Tracker.withComputation` call:
+
+```javascript
+
+Tracker.autorun(async function (computation) {
+  let asyncData = await someAsyncCall();
+  let links =
+    await Tracker.withComputation(computation, () => Meteor.users.find({}).fetch()); // will trigger reruns.
+});
+
+
+```
```

Comment:
@Grubba27 I still think that these comments should be in separate lines, so the reader won't have to scroll horizontally to see them, and the empty lines (both from the start and end of the second snippet) should be removed.

Reply:
I forgot this one. My bad. I will commit to `release-2.12` with these changes.